console/services/group_service.py

```diff
@@ -342,10 +342,10 @@ def get_multi_apps_all_info(self, app_ids, region, tenant_name, enterprise_id):
             if not app.get("allocate_mem"):
                 app["allocate_mem"] = 0
             for svc in app["service_list"]:
-                app["allocate_mem"] += svc.min_memory
+                app["allocate_mem"] += svc.min_memory * svc.min_node
                 if svc.status in ["running", "upgrade", "starting", "some_abnormal"]:
                     # if is running used_mem ++
-                    app["used_mem"] += svc.min_memory
+                    app["used_mem"] += svc.min_memory * svc.min_node
```

Comment:
This is still inconsistent with application-level memory usage.

Reply:
Restored, I will start a new pr to solve this problem